### DISCUSSION TITLE
azidentity update for vulnerability

### DISCUSSION THREAD

**Discussion by manish222agr** [argoproj/argo-cd#19412] [Q&A]
Hi,

I tried to pull the code for v2.10.9, and upgraded the azidentity from 1.1.0 to 1.6.0 manually using go get command. I am trying to do a docker build using DockerFile available in repo. But, the docker build is failing.

It does not give any reason to correctly locate the actual reason.

Please suggest why argocd does not build, and is it code backward compatibility issue. What other package upgrade or code update is required.

Thanks.

**Comment by crenshaw-dev** [comment]
Instead of running `docker build`, I'd recommend trying `make cli-local`. That'll run a go build and should make it easier to find any relevant error message.

**Reply by krishnakumar1990** [reply]
@manish222agr, were you able to resolve the issue. I have a similiar situation where we tried to upgrade from az1.10 to 1.6.0 and the build fails with no reable errors messages that points the details.
